lua/diffview/scene/views/diff/diff_view.lua

```diff
@@ -124,15 +125,17 @@ function DiffView:post_open()
 
   self.watcher = vim.loop.new_fs_poll()
   ---@diagnostic disable-next-line: unused-local
```

Comment:
Please move both of these lines inside the if clause you created. No need to create and hold on to a file descriptor we aren't using.